src/main/scala/com/gu/configraun/models/Configuration.scala

```diff
@@ -3,15 +3,15 @@ package com.gu.configraun.models
 import com.gu.configraun.Errors.{ ConfigraunError, ParamNotOfTypeError, ParamNotExistError }
 
 case class Configuration(underlying: Map[String, Param]) {
-  def getAsString(key: String): Either[ConfigraunError, String] = {
+  def getAsString(key: String) = {
```

Comment:
Why have you removed the return type of this method? All public methods should have return types.